daemon/metrics.go

```diff
@@ -141,7 +141,7 @@ func (p metricsPlugin) sock() string {
 }
 
 func (p metricsPlugin) sockBase() string {
-	return filepath.Join(p.BasePath(), "run", "balena")
+	return filepath.Join(p.BasePath(), "run", "balena[RENAME_THIS?]")
```

Comment:
What is this "metrics plugin"? :-) Should `balena` be renamed to `balenadctl` here?